engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -199,7 +199,7 @@ class Compiler(
           case None =>
             context.log(
               Level.SEVERE,
```

Comment:
This usage of logging is using `java.util.logging.Level` - are we supposed to use `java.util.logging.Logger` over `sl4j` in `engine/runtime`?

Reply:
Given the size of this PR and the time spent I'm inclined to do this unification at a next stage. We still need to do a few things, like structural logging so this PR is not the end of the road. And we also don't degrade - this jul vs slf4j distinction in format has been here before.